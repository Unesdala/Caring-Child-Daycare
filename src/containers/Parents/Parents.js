import React, { Component } from 'react';
import commonUtils from '../../components/lib/commonUtils';
import DefaultParentsContent from './ParentsContent';
import DefaultFooter from '../../components/Footer/Footer';

export class Parents extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Parent&apos;s Corner'); }

  render() {
    return (
      <div className="page-content">
        <DefaultParentsContent />
        <DefaultFooter />
      </div>
    );
  }
}

export default Parents;
