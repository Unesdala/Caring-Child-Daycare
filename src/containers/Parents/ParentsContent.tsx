import { Link } from 'react-router-dom';

const ParentsContent = (): JSX.Element => (
  <main className="parents-corner-main--content">

    <h1>Your link to our resources.</h1>

    <p>
      At Caring Child, we know parenting can be stressful. That&apos;s why we offer a range of resources to make
      getting through the day a little bit easier.
      Whether it be links to forms and papers that you&apos;ll need to register for our facilities, or general Virginia schools, the menu
      for your child&apos;s upcoming meals at our centers, or plans for emergencies, we hope to ease the stress of daily life even just a little bit.
    </p>

    <p>At Caring Child, both you and your child are in good hands.</p>

    <p>
      Feel free to browse our
      {' '}
      <Link to="/parents/resources">parent resources</Link>
      {' '}
      or take a look at what other parents
      {' '}
      <Link to="/parents/testimonials">have said</Link>
      {' '}
      about Caring Child Daycare.
    </p>

  </main>
);

export default ParentsContent;
