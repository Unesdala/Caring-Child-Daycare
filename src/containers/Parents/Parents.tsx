import ParentsContent from './ParentsContent';

export const Parents = (): JSX.Element => {
  document.title = 'Caring Child Daycare | Parent\'s Corner';  
  return (
    <div className="Site-content">
      <main className="parents-corner-main">
        <ParentsContent />
      </main>
    </div>
  );
};

export default Parents;
