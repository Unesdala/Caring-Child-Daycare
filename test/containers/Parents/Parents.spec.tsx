/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Parents } from 'src/containers/Parents/Parents';
import ParentsContent from 'src/containers/Parents/ParentsContent';

describe('Parents', () => {
  const wrapper = shallow(<Parents />);
  it('renders', () => {
    expect(wrapper.find(ParentsContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(ParentsContent).dive().find('main.parents-corner-main--content').exists()).toBe(true);
  });
  
});
