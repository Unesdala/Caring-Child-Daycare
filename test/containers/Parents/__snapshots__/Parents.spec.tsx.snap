// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parents renders 1`] = `ShallowWrapper {}`;
